### PR TITLE
Breaking changes added

### PR DESCRIPTION
**Pull request by effoeffi**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+    description: 'path of revised OpenAPI spec in YAML or JSON format'
     required: true
+  format:
+    description: 'Output format'
+    required: false
+    default: 'yaml'
+  breaking-only:
+    description: 'Display breaking changes only'
+    required: false
+    default: 'false'
```
Comment (reuvenharrison) [inline action.yml]: Better to use type: boolean

File: action.yml [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+  fail-on-diff:
+    description: 'Fail with exit code 1 if a difference is found'
+    required: false
+    default: 'false'
```
Comment (reuvenharrison) [inline action.yml]: Better to use type: boolean

File: action.yml (hunk 1 above)
Comment (effoeffi) [reply action.yml]: you're right. i started with that, but i do not find that github actions support boolean (only text).
take a look: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions

Comment (effoeffi) [reply action.yml]: anyway i changed to boolean and it's working ok

**Pull request merged**